### PR TITLE
Add script that generated untranslated json

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,16 +44,28 @@ This will prompt you to enter a language code and will generate your language fi
...
+- When PRs get merged they are immediately deployed live to GitHub pages. This makes maintenance really nice as there is no manual deploy step.
 
 The drawbacks of this approach:
 
 - Only the English (default) version is indexed by search engines
 - The English version briefly appears on page loads before the translated version is swapped in
+
+### Update Translations
+
+Not sure what pages are added after you translate it? Don't worry. :)
```
Comment (martinheidegger) [inline README.md]: "Are you not sure what translations is missing? Don't worry!"

Comment (marocchino) [inline README.md]: Apply in 7e4d9b7

**Comment by martinheidegger** [comment]
LGTM

**Comment by marocchino** [comment]
Thanks!

**Pull request merged**